src/app/rxjs/components/installation/installation.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-installation',
+  templateUrl: './installation.component.html',
+  styleUrls: ['./installation.component.scss']
+})
+export class InstallationComponent implements OnInit {
+  importPatching = `import { Observable} from 'rxjs/Observable';
+import 'rxjs/add/observable/of';
```

Comment:
Looks like this is misplaced code. Can you please remove these ?

Reply:
@ashwin-sureshkumar  These are not misplaced codes, they are part of the installation process, it was making the angular compiler complaining about having not recognisable code although it was string in template.
That's why I put them in a variable in the component. I can try to put them in the html template if that's not aligned with the guidelines.

Reply:
Don't worry about it I moved them to html, update coming soon